lightning-invoice/src/utils.rs

```diff
@@ -38,16 +38,20 @@ use sync::Mutex;
 ///   may be too long for QR code scanning. To fix this, `PhantomRouteHints::channels` may be pared
 ///   down
 ///
-/// `payment_hash` and `payment_secret` come from [`ChannelManager::create_inbound_payment`] or
+/// `payment_hash` and `payment_secret` can come from [`ChannelManager::create_inbound_payment`] or
 /// [`ChannelManager::create_inbound_payment_for_hash`]. These values can be retrieved from any
-/// participating node.
+/// participating node. Alternately, [`inbound_payment::create`] or
```

Comment:
tiny nit: if we mean that we can use these instead of the methods from a `ChannelManager`, then we'll need "Alternatively"
```suggestion
/// participating node. Alternatively, [`inbound_payment::create`] or
```